pkg/diagnose/diagnose.go

```diff
@@ -31,16 +38,100 @@ type Collector struct {
 // Cmd returns a CLI command
 func Cmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "diagnose",
```

Comment:
For backward compatibility, I think we should keep the diagnose command as an alias to `diagnostics collect`.

Reply:
We can, there is also the option to add a property `Deprecated` instead of an alias.